listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -239,6 +239,57 @@ def user_feed(user_name: str):
     }})
 
 
+@user_timeline_event_api_bp.route("/user/<user_name>/feed/events/delete", methods=['OPTIONS', 'POST'])
+@crossdomain(headers="Authorization, Content-Type")
+@ratelimit()
+def delete_feed_events(user_name):
+    '''
+    Delete those events from user's feed that they created themselves.
+    Supports deletion of recommendation, notification and pin events.
+    Along with the authorization token, post one of the following, according
+    to your need.
+    {
+        "event_type": "recording_recommendation",
+        "id": int
+    }
+    {
+        "event_type": "notification",
+        "id": int
+    }
+    :param user_name: The MusicBrainz ID of the user from whose timeline events are being deleted
+    :type user_name: ``str``
+    :statuscode 200: Successful deletion
+    :statuscode 400: Bad request, check ``response['error']`` for more details.
+    :statuscode 401: Unauthorized
+    :statuscode 404: User not found
+    :statuscode 500: API Internal Server Error
+    :resheader Content-Type: *application/json*
+    '''
+    # TODO: maybe implement logging once approved
```

Comment:
Logging for what? that an even was deleted? I don't think that is necessary.

Reply:
Alrighty, I will remove this comment :)